File: Java/daireAlan/DaireAlan.java
package daireAlan;

import java.util.Scanner;

public class DaireAlan {

	public static void main(String[] args) {
		
		Scanner s = new Scanner(System.in);
		
		System.out.print("Dairenin yarýçapýný giriniz : ");
		int yaricap = s.nextInt();
		System.out.print("Daire diliminin merkez açý ölçüsünü giriniz : ");
		int aci = s.nextInt();
		
		double pi = 3.14;
		double alan = (pi *(yaricap * yaricap) * aci) / 360;
		System.out.println("Daire diliminin alaný : "+ alan);
		
		
		
	}

}

File: NodeJs/DaireAlanı-Ödev1/daireAlan.js
const arguments = process.argv.slice(2);

function daireAlan(yaricap){

    let alan = Math.PI * yaricap * yaricap;
    
    console.log("Yarıçapı "+yaricap+" olan dairenin alanı : "+ alan);
}
daireAlan(arguments[0]*1);
